crates/bitwarden-crypto/src/keys/master_key.rs

```diff
@@ -138,19 +153,17 @@ impl MasterKey {
 
 /// Helper function to encrypt a user key with a master or pin key.
 pub(super) fn encrypt_user_key(
-    key: &KdfDerivedKeymaterial,
+    master_key: &KdfDerivedKeyMaterial,
     user_key: &SymmetricCryptoKey,
 ) -> Result<EncString> {
-    let mut userkey_bytes = user_key.to_vec();
-    let stretched_key = stretch_kdf_key(key)?;
-    let encrypted_userkey = EncString::encrypt_aes256_hmac(&userkey_bytes, &stretched_key);
-    userkey_bytes.zeroize();
-    encrypted_userkey
+    let stretched_master_key = stretch_kdf_key(master_key)?;
+    let userkey_bytes = Zeroizing::new(user_key.to_vec());
+    EncString::encrypt_aes256_hmac(&userkey_bytes, &stretched_master_key)
```

Comment:
We generally refer to this as `user_key` not `userkey`.
```suggestion
    let user_key_bytes = Zeroizing::new(user_key.to_vec());
    EncString::encrypt_aes256_hmac(&user_key_bytes, &stretched_master_key)
```